src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public abstract class Machine {
+    public void doWork() {
+    }
+
+    public void stopWork() {
+    }

```

Comment:
make methods `abstract` if they consist of any logic. 
```suggestion
    public void doWork() {
    }

    public void stopWork() {
    }

```